src/component/feature/form/index.jsx

```diff
@@ -35,29 +26,12 @@ class AddFeatureToggleComponent extends Component {
             editmode = false,
         } = this.props;
 
-        const {
-            name, // eslint-disable-line
-            nameError,
-            description,
-            enabled,
-        } = input;
+        const { description, enabled } = input;
         const configuredStrategies = input.strategies || [];
 
         return (
             <form onSubmit={onSubmit(input)}>
                 <section style={{ padding: '16px' }}>
-                    <Textfield
```

Comment:
This component is also used when creating a toggle. By removing the name field there is no way of defining the name of a toggle when creating a new one.

Reply:
...on that note: Having the same component for create and edit might not be the best approach if we end up with a lot of if/else clutter imho.

Reply:
oops that's bad indeed :)) Let me put it back. Indeed we could do conditional. Let's see how it looks like.